Source/Tests/Paths/ConfigDirPathTests.cs

```diff
@@ -2,25 +2,34 @@ namespace Bloodmasters.Tests.Paths;
 
 public class ConfigDirPathTests
 {
-    [Fact(DisplayName = "ConfigDirPath should be named 'Debug' in dev mode and 'Config' otherwise")]
+    [Fact(DisplayName = "ConfigDirPath should be named 'Config' in release mode")]
     public void ConfigDirPathShouldBeNamedDebugInDevModeAndConfigOtherwise()
+    {
+        var paths = new CodeImp.Bloodmasters.Paths();
+        var dirName = Path.GetFileName(paths.ConfigDirPath);
+
+        // Assert
+        Assert.Equal("Config",dirName);
+    }
+
+    [Fact(DisplayName = "ConfigDirPath should be named 'Debug' in dev mode")]
+    public void ConfigDirPathShouldBeNamedDebugInDevMode()
     {
         // Arrange
-        var dirName = Path.GetFileName(CodeImp.Bloodmasters.Paths.ConfigDirPath);
+        var paths = new CodeImp.Bloodmasters.Paths(isDev: true);
+        var dirName = Path.GetFileName(paths.ConfigDirPath);
 
         // Assert
-        Assert.Equal(
-            CodeImp.Bloodmasters.Paths.IsDevModeBuild ? "Debug" : "Config",
-            dirName);
+        Assert.Equal("Debug", dirName);
     }
```

Comment:
This whole test is incorrect. `isDev` flag should not control whether `Debug` or `Release` is chosen.

The "dev mode" is when you build the game from sources, either in Debug or Release build configuration.